### PR TITLE
Ammo selection.

### PR DESCRIPTION
**Pull request by tclaar**
Allow ammo type selection using number keys.
Also fix an issue where firing the slingshot when an infinite ammo type is selected crashes the game.

### CODE REVIEW

File: js/scripts/util.js [hunk 1]
```diff
@@ -305,6 +305,15 @@ const EVENT_HANDLERS = {
             case "KeyF":
                 GAME.user.eatFood = true; // this should disable automatically after food is eaten
                 break;
+            case "Digit1":
```
Comment (NathanHinthorne) [inline js/scripts/util.js:308]: Looks good

**Review by NathanHinthorne — APPROVED** [review]

File: js/scripts/util.js [hunk 2]
```diff
@@ -305,6 +305,15 @@ const EVENT_HANDLERS = {
...
+            case "Digit2":
```
Comment (calebkrauter) [inline js/scripts/util.js:309]: Using the word Digit as a case seems a bit unclear. It took me a moment to remember what was happening to know what this is used for. What about some name such as numKey1 etc?

**Review by calebkrauter — APPROVED** [review]
Looks good.

File: js/scripts/util.js (hunk 2 above)
Comment (tclaar) [reply js/scripts/util.js:309]: These are JS key codes, just like `"KeyA"`, `"KeyB"`, etc. and can't be changed. Check this out for more info: https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values.

**Pull request merged**